### ISSUE TITLE
Questions about papi catalog entry

### ISSUE THREAD

**Issue by bernhold** [cass-community/cass-community.github.io#35]
Repository and Downloads have the same URL, which appears to be a Repository.  Is there a relevant Downloads link?  Perhaps https://github.com/icl-utk-edu/papi/releases ?

Modified `description` to remove period.

**Comment by jagode** [comment]
This is correct---Repository and Downloads refer to the same URL. Please see the attached screenshot of the **PAPI Repository** for reference.

- **Option (1)** (highlighted in red in the screenshot) is the only place to download _the latest, most up-to-date version of PAPI_ from the master branch in the Repository, which is generally recommended.
- Alternatively, **Option (2)** (also highlighted in red in the screenshot) allows users to download the latest _release version of PAPI_.

All other release versions of PAPI can be downloaded from the Repository Wiki: PAPI Releases.

### SOLUTION:

If possible, we could list **two** URLs under "Downloads":
```
https://github.com/icl-utk-edu/papi
https://github.com/icl-utk-edu/papi/wiki/PAPI-Releases
```

However, if only one URL is desired, then the PAPI Repository is the appropriate choice:
`https://github.com/icl-utk-edu/papi `


<img width="1322" alt="papi-repo" src="https://github.com/user-attachments/assets/a19c45cc-b7f0-4647-bbf1-a7e1c2ebdba2" />
